src/utils/sources-tree/getURL.js

```diff
@@ -10,6 +10,7 @@ import { getUnicodeHostname, getUnicodeUrlPath } from "devtools-modules";
 import type { Source } from "../../types";
```

Comment:
We don't need any of these changes anymore; these were left over from your first attempt.